backend/core/connection_crud.py

```
@@ -46,4 +46,11 @@ def get_connections(db: Session, user_id: int):
     return db.query(Connection).filter(
         (Connection.user_id == user_id) | (Connection.friend_id == user_id),
         Connection.status == ConnectionStatus.ACCEPTED
-    ).all()
\ No newline at end of file
+    ).all()
+
+def get_pending_requests(db: Session, user_id: int):
```

Ruff (0.8.2)

Missing return type annotation for public function `get_pending_requests`

(ANN201)